### PR TITLE
Migration guide update

### PR DESCRIPTION
**Pull request by godai78**
Migration guide fixes:

*   new item added
*   items expanded
*   linking fixed for good

### CODE REVIEW

**Comment by godai78** [comment]
@ckeditor/qa-team - guys, can You please have a look at it?

File: docs/builds/guides/migration/migration-from-ckeditor-4.md [hunk 1]
```diff
@@ -420,264 +420,264 @@ Note: The number of options was reduced on purpose. We understood that configuri
...
-			<td>N/A. The magic line feature itself is <a href="https://github.com/ckeditor/ckeditor5/issues/407" target="_blank" rel="noopener">planned</a>.</td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-magicline_color">magicline_color</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-magicline_everywhere">magicline_everywhere</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-magicline_holdDistance">magicline_holdDistance</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-magicline_keystrokeNext">magicline_keystrokeNext</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-magicline_keystrokePrevious">magicline_keystrokePrevious</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-magicline_tabuList">magicline_tabuList</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-magicline_triggerOffset">magicline_triggerOffset</a></td>
+			<td>This features is managed by the {@link module:widget/widgettypearound~WidgetTypeAround <code>WidgetTypeAround</code>} class.</td>
```
Comment (AnnaTomanek) [inline docs/builds/guides/migration/migration-from-ckeditor-4.md:487]:
```suggestion
            <td>This functionality is covered by the {@link module:widget/widgettypearound~WidgetTypeAround <code>WidgetTypeAround</code>} plugin that allows users to type around widgets where normally it is impossible to place the caret due to limitations of web browsers.</td>
```

File: docs/builds/guides/migration/migration-from-ckeditor-4.md [hunk 2]
```diff
@@ -420,264 +420,264 @@ Note: The number of options was reduced on purpose. We understood that configuri
...
 		</tr>
 		<tr>
-			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates">templates</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates_files">templates_files</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates_replaceContent">templates_replaceContent</a></td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates">templates</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates_files">templates_files</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates_replaceContent">templates_replaceContent</a></td>
 			<td>N/A</td>
 		</tr>
 		<tr>
-			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-title">title</a></td>
-			<td>N/A</td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-title">title</a></td>
```
Comment (AnnaTomanek) [inline docs/builds/guides/migration/migration-from-ckeditor-4.md:635]: This is not the same feature. Compare the API docs and guide of these two to check this out.

**Review by AnnaTomanek — CHANGES_REQUESTED** [review]

File: docs/builds/guides/migration/migration-from-ckeditor-4.md (hunk 2 above)
Comment (godai78) [reply docs/builds/guides/migration/migration-from-ckeditor-4.md:635]:
```suggestion
			<td>N/A</td>
```

**Review by AnnaTomanek — APPROVED** [review]

**Pull request merged**